corehq/apps/data_dictionary/tests/test_view.py

```diff
@@ -245,3 +245,41 @@ def test_no_view_access(self):
         self.client.login(username='no_data_dict@ex.com', password='foobar')
         response = self.client.get(self.url)
         self.assertEqual(response.status_code, 403)
+
+
+@flag_enabled('DATA_DICTIONARY')
+class TestDeprecateOrRestoreCaseTypeView(TestCase):
```

Comment:
It's for the `deprecate_or_restore_case_type` view. Since the only difference between deprecating and restoring a case type is changing the `is_deprecated` property, I thought if we can test setting the property to `True`, then we can assume it will work for setting it to `False` as well.

Do you think we should rather not make this assumption and instead put in a test for it?

Reply:
I think that's fine, thanks for the clarification!